### ISSUE TITLE
Sudo Clink version mismatch

### ISSUE THREAD

**Issue by HeyItsJono** [vladimir-kotikov/clink-completions#168]
Hi there,
Despite having the latest version of clink (Clink v1.4.23.5fc3fc), the sudo clink completion provided with this repo seems to throw a version mismatch error when I begin to type `sudo` in my command prompt. I get the specific error:

    gsudo.lua argmatcher requires a newer version of Clink; please upgrade.
    sudo.lua argmatcher requires a newer version of Clink; please upgrade.

I assume this is the check for `clink_version.supports_argmatcher_chaincommand` failing, but I can't imagine why this would be the case on the latest version. For what it's worth I do have `sudo` aliased to `gsudo` if it makes a difference. Any help you could offer would be much appreciated thanks!

**Comment by HeyItsJono** [comment]
Thanks for the quick response! Here's the output for the Clink version:

    version:
      version           1.4.23.5fc3fc
      binaries          USERPROFILE\AppData\Roaming\Cmder\vendor\clink
    session:
      session           39428
      profile           USERPROFILE\AppData\Roaming\Cmder\config
      log               USERPROFILE\AppData\Roaming\Cmder\config\clink.log
      settings          USERPROFILE\AppData\Roaming\Cmder\config\clink_settings
      history           USERPROFILE\AppData\Roaming\Cmder\config\clink_history
      scripts           USERPROFILE\AppData\Roaming\Cmder\vendor ; USERPROFILE\AppData\Roaming\Cmder\vendor\clink ; USERPROFILE\AppData\Roaming\Cmder\config ; .;USERPROFILE\AppData\Roaming\Cmder\vendor\more-clink-completions;USERPROFILE\AppData\Roaming\Cmder\config\clink-gizmos;USERPROFILE\AppData\Roaming\Cmder\config;USERPROFILE\AppData\Roaming\Cmder\config\clink-completions
      inputrc           USERPROFILE\AppData\Roaming\Cmder\config\.inputrc
    
    (Use a numeric argument for additional diagnostics; e.g. press Alt+1 first.)


I installed clink-completions via `git clone` to `USERPROFILE\AppData\Roaming\Cmder\config\clink-completions`, and I update via `git pull origin master`.

**Comment by HeyItsJono** [comment]
Great catch with the `.` directory thank you, that's removed now.

I just backed `vendor\clink-completions\` up to a 7z archive (just in case) and deleted everything in the folder except the `.init.lua` file, which I modified to point to the `config\clink-completions\modules` folder instead (since there is now no longer a `vendor\clink-completions\modules` folder.

I tried just deleting the whole folder initially but it seems some part of the Cmder initialization process explicitly calls `vendor\clink-completions\.init.lua` and throws an error each time a shell starts if it doesn't exist:

`cannot open USERPROFILE\Roaming\Cmder/vendor/clink-completions/.init.lua: No such file or directory`

I'm still looking for what actually calls this so I can disable it. I wish there was a cleaner way to update clink-completions without having to modify the initial Cmder distribution, I think the way things are at the moment I'm just going to have to redo this process every time I update Cmder. 

Windows doesn't resync the files after deletion because I have no Roaming profile enabled; whilst these files are stored in `AppData\Roaming` by the preference of the Cmder devs, my Windows profile doesn't sync to any server, so my `AppData\Roaming` folder is functionally identical to the `AppData\Local` folder (at least, this is my understanding).
